CHANGES.rst

```diff
@@ -17,6 +17,7 @@ Breaking changes:
 New features:
 
 - Create GitHub releases for each tag.
+- Allow using ``zoneinfo`` as a timezone implementation.
```

Comment:
I'd like to have a little bit more elaborate Changelog entry.
E.g. do need people update their code?
Or are there no backwards incompatible changes?
Was anything else done except for the zoneinfo change?

Reply:
I edited this in 6478c12 so that I can also create a v5.0.13 release.